ptf/.pylintrc

```diff
@@ -0,0 +1,591 @@
+[MASTER]
```

Comment:
Let's exclude the pylintrc file for now

Reply:
I have just added this file for our internal purposes